app/assets/stylesheets/elements_table.scss

```diff
@@ -90,22 +90,6 @@ th.drag {
   }
 }
 
-/* .panel-title div:empty:after {
-  content: ".";
-  visibility: hidden;
-} */
-
-
-/*.number-shown-select {
-  float: right !important;
-  text-align: right;
-  width: 55% !important;
-}*/
-
-/*.element-svg-checkbox {
-  margin: 0px 0 0;
-}*/
-
```

Comment:
@mekkyz Have these been removed due to no usage within the code?

Reply:
Yes, they were commented out with no reason.